migrations/Version202608031750001488_taoAdvancedSearch.php

```
@@ -0,0 +1,37 @@
+    public function up(Schema $schema): void
+    {
+        $script = new RegisterItemCommentElasticsearchAdapter();
+        $script->setServiceLocator($this->getServiceLocator());
+        $script([]);
+
+        $this->addReport(
+            Report::createSuccess('Item Comment persistence adapter switched to Elasticsearch')
+        );
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)